string lamp_class

uint32 fps
uint16 xmin
uint16 ymin
uint16 xmax
uint16 ymax

